janus_server/src/aggregator.rs

```diff
@@ -1022,6 +1001,108 @@ impl VdafOps {
             .await?)
     }
 
+    async fn handle_collect(
+        &self,
+        datastore: &Datastore,
+        task: &Task,
+        collect_req: &CollectReq,
+    ) -> Result<Uuid, Error> {
+        match self {
+            VdafOps::Prio3Aes128Count(_, _) => {
+                Self::handle_collect_generic::<Prio3Aes128Count, FieldError>(
```

Comment:
Q: is Rust smart enough to infer the second generic parameter if we specify the first? (e.g. does compilation succeed if we specify this as `<Prio3Aes128Count, _>`).

I'd like it even better if this took a single generic parameter that was the VDAF type since it's not immediately clear what the error type's relation/purpose is, but I think that might make the trait bounds on `handle_collect_generic` pretty confusing.

Reply:
You were right, Rust can figure it out if we don't specify E! It'd be nice if there was a clippy warning or something for "unnecessary type parameter".